news/2017-10-20-scorewatch-october-week-1.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Scorewatch: October Week 1"
+date: 2017-10-20 15:00:00 +0000
+---
+
+October, the spooky month, has arrived and scores are being set more than ever! We got more 700pp scores, a rustbell classic and more in the first October Scorewatch!
+
+![](/wiki/shared/news/2017-10-20-scorewatch-october-week-1/banner.jpg)
+
+[Bubbleman](https://osu.ppy.sh/u/5182050) is taking his limits to the next level as he snipes a year old score on [MY FIRST STORY - ALONE](https://osu.ppy.sh/b/861381), taking an amazing 99.91% HDHR FC to grab 527pp. To think that he nearly SSed it (got the 1x 100 on the 2nd last note) is amazing enough!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/yWXkzn1roPE" frameborder="0" allowfullscreen></iframe></br>
+
+[Rafis](https://osu.ppy.sh/u/2558286) has sniped hvick225’s year old score on [Hana - Sakura no Uta](https://osu.ppy.sh/b/827488)! This is Rafis’ 11th 700pp score as he sets an impressive 99.84% HDDT FC on the map, earning 704pp. Get ready for some intense jumping and streaming!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/wdTxovl8rMU" frameborder="0" allowfullscreen></iframe></br>
+
+[Cookiezi](https://osu.ppy.sh/u/124493) has reached the 14,000pp mark with an amazing 99.25% HDHR 1 miss score on [Linkin Park - Guilty All The Same (feat. Rakim)](https://osu.ppy.sh/b/1187302?m=0) to earn 726pp. Enjoy 6 minutes of Linkin Park!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/HDfWRLlqhY8" frameborder="0" allowfullscreen></iframe></br>
+
+[cheewee10](https://osu.ppy.sh/u/4477497) sniped WindyS to the number one spot on [gmtn. (witch's slave) - furioso melodia](https://osu.ppy.sh/b/687730) on the 7K difficulty with a painful 99.99% (1 200!) SD FC, earning himself 390pp. Imagine if he put the PF mod instead.
```

Comment:
sub PF mod for Perfect, and SD for Sudden Death, since they're moderately uncommon as far as highscores go